src/routes/api/db-migrations/db-migration-down.ts

```diff
@@ -0,0 +1,59 @@
+import { Request, Response } from "express";
+import { getLogger } from "config/logger";
+import safeJsonStringify from "safe-json-stringify";
+import { sequelize } from "models/sequelize";
+import { QueryTypes } from "sequelize";
+import { getTargetScript, validateScriptLocally, startDBMigration, DBMigrationType } from "./db-migration-utils";
+
+const logger = getLogger("DBMigrationDown");
+
+export const DBMigrationDown = async (req: Request, res: Response): Promise<void> => {
+
+	try {
+
+		const targetScript = getTargetScript(req);
+		logger.info(`Received DB target script to mgiration DOWN: ${targetScript}`);
+
+		await validateScriptLocally(targetScript);
+		await validateScriptAgainstDB(targetScript);
+		const { isSuccess, stdout, stderr } = await startDBMigration(targetScript, DBMigrationType.DOWN);
+
+		res.status(isSuccess ? 200: 500).send(`
```

Comment:
Would JSON be easier?

Reply:
Tried that locally just now, realise since the major consume is human/developer who run this script, the following screenshot (with json output) is not very user friendly, compare to the one in this PR's description.

thoughts?

![image](https://user-images.githubusercontent.com/105693507/191137937-2697d694-c8e4-4afa-9e48-7ebd3a490fd0.png)